Prac_07/KivyDemos-master/dynamic_labels.kv

```diff
@@ -0,0 +1,4 @@
+BoxLayout:
+    orientation: 'vertical'
+    Label:
```

Comment:
For this to dynamically work, you need to give a widget an id, but not the root widget. Have a look at the dynamic_widgets.kv but focus only on the top bits, which is flagged with an id.